### PR TITLE
Table component builder

### PR DESCRIPTION
**Pull request by mariarivera30**
Next adding test

### CODE REVIEW

File: static/src/components/builder/store/modules/table.js [hunk 1]
```diff
@@ -5,15 +5,12 @@ const prop = (value, isVariable) => {
...
+export const getColumnObject = (id) => {
     return {
         name: '',
         header: '',
         component: 'plain-text',
-        filterable: false,
-        sortable: false,
-        isColumnId: false,
-        hideColumnId: false
+        id: 'Columns ' + id
```
Comment (fiorda) [inline static/src/components/builder/store/modules/table.js]: you can use template strings

File: static/src/components/builder/components/TextInput/TextInputForm.vue [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
 export default {
     name: 'TextInputForm',
-    components: { FormCommons },
-    data () {
-        return { }
-    },
+    computed: {
+        form: {
+            get () {
+                const getFormType = types['GET_' + this.$route.params.componentName + '_FORM']
```
Comment (fiorda) [inline static/src/components/builder/components/TextInput/TextInputForm.vue]: string templates here too

File: static/src/components/builder/store/modules/table.js (hunk 1 above)
Comment (mariarivera30) [reply static/src/components/builder/store/modules/table.js]: yes, and also replaced all of them by templating

File: static/src/components/builder/components/Questions.vue [hunk 3]
```diff
@@ -91,8 +147,31 @@ export default {
...
+        this.loading = true
+        this.getTaskPresenterTemplate()
+    },
+    methods: {
+        async getTaskPresenterTemplate () {
+            await fetch(`${window.location.pathname}?template=helper-components`, {
+                method: 'GET',
+                headers: {
+                    'content-type': 'application/json',
+                    'X-CSRFToken': this.csrfToken
```
Comment (fiorda) [inline static/src/components/builder/components/Questions.vue]: I don't think you need the CSRF token here

File: static/src/components/builder/components/Questions.vue [hunk 4]
```diff
@@ -91,8 +147,31 @@ export default {
...
+                }
+            }).then((res) => {
```
Comment (fiorda) [inline static/src/components/builder/components/Questions.vue]: can you do `const res = await fetch` instead of using the promise API

**Pull request merged**